lib/rmagick_internal.rb

```diff
@@ -8,6 +8,16 @@
 #               to the classes.
 #==============================================================================
 
+if RUBY_PLATFORM =~ /mingw/i
```

Comment:
I see a couple of places in RMagick that also check for `mswin`. Do we need to handle that as well, or is this only a problem in mingw?

Reply:
RubyInstaller only provides the `MinGW` environment so far.
The problem that I want to fix in this Pull Request is occurred in RubyInstaller environment.